app/src/main/java/com/example/marvel_app/home/HomeFragment.kt

```diff
@@ -0,0 +1,33 @@
+package com.example.marvel_app.home
+
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.lifecycle.ViewModelProvider
+import androidx.recyclerview.widget.GridLayoutManager
+import com.example.marvel_app.ComicsAdapter
+import com.example.marvel_app.databinding.FragmentHomeBinding
+
+class HomeFragment : Fragment() {
+
+    private val viewModel: HomeViewModel by lazy {
+        ViewModelProvider(this).get(HomeViewModel::class.java)
+    }
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        // Inflate the layout for this fragment
+        val binding = FragmentHomeBinding.inflate(inflater)
+        binding.lifecycleOwner = this
+        binding.viewModel = viewModel
+
+        val manager = GridLayoutManager(activity,1)
+        binding.comicsListHome.layoutManager = manager
+        binding.comicsListHome.adapter = ComicsAdapter()
```

Comment:
If you want to apply a few things to one object you can use apply syntax:
binding.comicsListHome.apply {
   layoutManager = manager
  adapter = ComicsAdapter()
}